controllers/propagator/replicatepolicy_pb_eventHandler.go

```diff
@@ -0,0 +1,94 @@
+package propagator
+
+import (
+	"context"
+
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	"open-cluster-management.io/governance-policy-propagator/controllers/common"
+)
+
+// EnqueueReqsFromMapFuncByBinding maps a PlacementBinding to the targeted RepPolicies that are either directly in its
+// subjects list, or are in a PolicySet which is a subject of this PlacementBinding.
+func EnqueueReqsFromMapFuncByBinding(c client.Client) handler.EventHandler {
```

Comment:
Maybe we can rename these types and functions to something like `HandlerForBinding`

Reply:
`EnqueueReqsFromMapFuncByBinding` -> `HandlerForBinding` ?